### ISSUE TITLE
Repeater falls over if selectWoo is not available

### ISSUE THREAD

**Issue by mikedevelops** [jadu/pulsar#1150]
**Describe the bug**
The Repeater falls over when instantiated in an environment that does not have selectWoo.

The repeater has to do some work on creating unique attributes on inputs, there is a special case for select2/woo because of all the HTML that comes along with those plugins. We should be able to just check it exists before attempting to get/set things.

**To Reproduce**
Remove SelectWoo from Pulsar's exported modules

**Expected behavior**
Without SelectWoo everything should be fine.

**Screenshots**
![Screenshot 2020-02-26 at 16 01 51](https://user-images.githubusercontent.com/9677023/75366077-16eacc00-58b6-11ea-84a4-9108847e6a63.png)

**Desktop (please complete the following information):**
Not a domain specific problem.

---

Fix this too! 

```
const $preview = $previewRoot.find(`[data-repeater-preview-id="${repeaterId}"`);
const $edit = $previewRoot.find(`[data-repeater-edit-id="${repeaterId}"`);
const $saved = $previewRoot.find(`[data-repeater-saved-data-id="${repeaterId}"`);
```

**Comment by mikedevelops** [comment]
Culprit looks to be here 
```       
js/Repeater/RepeaterComponent.js:339        

// Create unique IDs for selectWoo elements
this.uniqueIdService.uniquifySelectWoo(group);
```

We can either check for selectWoo inside the repeater component itself here, or check inside the uniqify service itself and just do a noop if it ins't. I think I probably prefer the first approach, should be a few lines change either way 🙏

**Comment by jamesjacobs** [comment]
Fixed by #1238
